contracts/handler/CompatibilityFallbackHandler.sol

```diff
@@ -3,7 +3,7 @@ pragma solidity >=0.7.0 <0.9.0;
 
 import {TokenCallbackHandler} from "./TokenCallbackHandler.sol";
 import {ISignatureValidator} from "../interfaces/ISignatureValidator.sol";
-import {Safe} from "../Safe.sol";
+import {ISafe} from "../interfaces/ISafe.sol";
```

Comment:
Could we import it as `Safe`? then it would not trigger any additional code changes (same was done for `Enum`)

Reply:
I think keeping it with an `I` is significant to notify the user that it is an `interface` and to follow a usual standard of interfaces. Also, using the same name will lead to `HardhatError: HH701: There are multiple artifacts for contract "Safe", please use a fully qualified name.` in tests (though it could be avoided by using the entire path to the contract I believe).

For `Enum`, the decision was taken to avoid the changes within the contract like `Enum.Operation operation` -> `IEnum.Operation operation`